src/IO/ADIOS/ADIOS1IOHandler.cpp

```diff
@@ -301,6 +293,19 @@ ADIOS1IOHandlerImpl::open_read( std::string const& name )
     return f;
 }
 
+int64_t
+ADIOS1IOHandlerImpl::initialize_group(std::string const &name)
+{
+    int status;
+    int64_t group;
+    ADIOS_STATISTICS_FLAG noStatistics = adios_stat_no;
```

Comment:
just a note: these statistics are quite interesting, but I never evaluated the RT costs in our use cases, so we turned them off by default.

if we could control from the outside we could evaluate them.

maybe good for an issue.